NPE in WorkerModule

I checked with the latest Docker image
```
com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error in custom provider, java.lang.NullPointerException: Job store can't be null
  at org.datatransferproject.transfer.WorkerModule.getExporter(WorkerModule.java:122)
  at org.datatransferproject.transfer.WorkerModule.getExporter(WorkerModule.java:122)
  while locating org.datatransferproject.spi.transfer.provider.Exporter

1 error
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1028)
	at org.datatransferproject.transfer.CallableExporter.call(CallableExporter.java:49)
	at org.datatransferproject.transfer.CallableExporter.call(CallableExporter.java:31)
	at org.datatransferproject.types.transfer.retry.RetryingCallable.call(RetryingCallable.java:66)
	at org.datatransferproject.transfer.PortabilityInMemoryDataCopier.copyHelper(PortabilityInMemoryDataCopier.java:109)
	at org.datatransferproject.transfer.PortabilityInMemoryDataCopier.copy(PortabilityInMemoryDataCopier.java:74)
	at org.datatransferproject.transfer.JobProcessor.processJob(JobProcessor.java:110)
	at org.datatransferproject.transfer.Worker.doWork(Worker.java:33)
	at org.datatransferproject.transfer.WorkerMain.poll(WorkerMain.java:142)
	at org.datatransferproject.bootstrap.vm.SingleVMMain$WorkerRunner.run(SingleVMMain.java:109)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException: Job store can't be null
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:889)
	at org.datatransferproject.datatransfer.google.drive.DriveImporter.<init>(DriveImporter.java:44)
	at org.datatransferproject.datatransfer.google.GoogleTransferExtension.initialize(GoogleTransferExtension.java:99)
	at org.datatransferproject.transfer.WorkerModule.getExporter(WorkerModule.java:124)
	at org.datatransferproject.transfer.WorkerModule$$FastClassByGuice$$26239bb9.invoke(<generated>)
	at com.google.inject.internal.ProviderMethod$FastClassProviderMethod.doProvision(ProviderMethod.java:264)
	at com.google.inject.internal.ProviderMethod$Factory.provision(ProviderMethod.java:401)
	at com.google.inject.internal.ProviderMethod$Factory.get(ProviderMethod.java:376)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:194)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
	at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1019)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1015)
	... 14 more
```

--- a/portability-api/src/main/java/org/datatransferproject/api/ApiMain.java
+++ b/portability-api/src/main/java/org/datatransferproject/api/ApiMain.java
@@ -38,6 +38,7 @@ import org.datatransferproject.spi.api.transport.TransportBinder;
 import org.datatransferproject.spi.cloud.extension.CloudExtension;
 import org.datatransferproject.spi.cloud.storage.AppCredentialStore;
 import org.datatransferproject.spi.cloud.storage.JobStore;
+import org.datatransferproject.spi.cloud.storage.TemporaryPerJobDataStore;
 import org.datatransferproject.spi.service.extension.ServiceExtension;
 import org.datatransferproject.types.transfer.auth.TokenAuthData;
 import org.datatransferproject.types.transfer.auth.TokenSecretAuthData;
@@ -130,6 +131,7 @@ public class ApiMain {
     // Needed for GoogleAuthServiceExtension
     extensionContext.registerService(HttpTransport.class, new NetHttpTransport());
     extensionContext.registerService(JobStore.class, cloudExtension.getJobStore());
+    extensionContext.registerService(TemporaryPerJobDataStore.class, cloudExtension.getJobStore());
     extensionContext.registerService(
         AppCredentialStore.class, cloudExtension.getAppCredentialStore());
 

--- a/portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java
+++ b/portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerMain.java
@@ -34,6 +34,7 @@ import org.datatransferproject.security.SymmetricKeyGenerator;
 import org.datatransferproject.spi.cloud.extension.CloudExtension;
 import org.datatransferproject.spi.cloud.storage.AppCredentialStore;
 import org.datatransferproject.spi.cloud.storage.JobStore;
+import org.datatransferproject.spi.cloud.storage.TemporaryPerJobDataStore;
 import org.datatransferproject.spi.service.extension.ServiceExtension;
 import org.datatransferproject.spi.transfer.extension.TransferExtension;
 import org.datatransferproject.spi.transfer.hooks.JobHooks;
@@ -94,6 +95,7 @@ public class WorkerMain {
 
     JobStore jobStore = cloudExtension.getJobStore();
     extensionContext.registerService(JobStore.class, jobStore);
+    extensionContext.registerService(TemporaryPerJobDataStore.class, jobStore);
 
     AppCredentialStore appCredentialStore = cloudExtension.getAppCredentialStore();
     extensionContext.registerService(AppCredentialStore.class, appCredentialStore);

